app/assets/javascripts/components/user_profiles/by_students.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+const ByStudents = React.createClass({
```

Comment:
I think this component's name should include 'stats', like `ByStudentsStats`, to more clearly indicate what it's for.